Cannot build 0c0686c and later on Ubuntu 12.04 (3.11.0-15 kernel)

The changes to fix Travis in f80e846dce4ac38403200917aec4b463c5065cb5 partially addresses the issues I hit on Friday with the new features in 0c0686c, but not completely.

```
[ 69%] Building C object CMakeFiles/seccomp.dir/src/test/seccomp.c.o
/usr/bin/cc   -pthread -O0 -g3 -Wall -Werror -Wstrict-prototypes   -I/home/demo/rr/rr/include -I/home/demo/rr/obj    -o CMakeFiles/seccomp.dir/src/test/seccomp.c.o   -c /home/demo/rr/rr/src/test/seccomp.c
/home/demo/rr/rr/src/test/seccomp.c: In function ‘install_filter’:
/home/demo/rr/rr/src/test/seccomp.c:25:29: error: ‘SECCOMP_SET_MODE_FILTER’ undeclared (first use in this function)
/home/demo/rr/rr/src/test/seccomp.c:25:29: note: each undeclared identifier is reported only once for each function it appears in
```

Adding `#include "kernel_supplement.h"` to `src/test/seccomp.c` doesn't quite fix it--`source` is not on the include path. Should `kernel_supplement.h` get moved to the `include` directory?

--- a/src/test/rrutil.h
+++ b/src/test/rrutil.h
@@ -184,4 +184,11 @@ inline static void free_guard(size_t size, void* p) {
 #define VERIFY_GUARD(p) verify_guard(sizeof(*p), p)
 #define FREE_GUARD(p) free_guard(sizeof(*p), p)
 
+#ifndef SECCOMP_SET_MODE_STRICT
+#define SECCOMP_SET_MODE_STRICT 0
+#endif
+#ifndef SECCOMP_SET_MODE_FILTER
+#define SECCOMP_SET_MODE_FILTER 1
+#endif
+
 #endif /* RRUTIL_H */

